dashboard/lib/pd/foorm/README.md

```diff
@@ -0,0 +1,187 @@
+#Foorm Survey Pipeline
```

Comment:
👏 for README! Nits -- I think you need a space between # and Foorm to get markdown formatting. Also possible to put method calls in backticks?

Reply:
fun fact: the RubyMine markdown previewer works without putting a space between # and title, but github does not! Will also put the methods in backticks, good callout